consensus/parlia/parlia.go

```diff
@@ -820,6 +820,11 @@ func (p *Parlia) Delay(chain consensus.ChainReader, header *types.Header, leftOv
 	return &delay
 }
 
+// ExtraSeal returns fixed number of extra-data suffix bytes reserved for signer seal
```

Comment:
What is this for?

Reply:
It's used to get signature from `header.Extra`